src/app/test-record/test-record.container.ts

```diff
@@ -1,60 +1,100 @@
 import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
 import { Observable } from 'rxjs';
-import { Store, select } from '@ngrx/store';
+import { Store } from '@ngrx/store';
 import { IAppState } from '@app/store/state/app.state';
-import { selectTestTypeById } from '@app/store/selectors/VehicleTestResultModel.selectors';
-import {ActivatedRoute} from '@angular/router';
-import {TestResultModel} from '@app/models/test-result.model';
-import {TestType} from '@app/models/test.type';
-import {initAll} from 'govuk-frontend';
-import {TestRecordMapper} from '@app/test-record/test-record.mapper';
+import {
+  getTestViewState,
+  selectTestTypeById
+} from '@app/store/selectors/VehicleTestResultModel.selectors';
+import { ActivatedRoute } from '@angular/router';
+import { TestRecordMapper, TestTypesApplicable } from '@app/test-record/test-record.mapper';
+import { Preparer } from '@app/models/preparer';
+import { TestRecordTestType } from '@app/models/test-record-test-type';
+import { VIEW_STATE } from '@app/app.enums';
+import { TestResultModel } from '@app/models/test-result.model';
+import {
+  SetTestViewState,
+  UpdateTestResult
+} from '@app/store/actions/VehicleTestResultModel.actions';
+import { TestStation } from '@app/models/test-station';
+import { getPreparers, getTestStations } from '@app/store/selectors/ReferenceData.selectors';
+import { TestResultTestTypeNumber } from '@app/models/test-result-test-type-number';
 
 @Component({
   selector: 'vtm-test-record-container',
   template: `
-    <ng-container *ngIf="testRecordObservable$ | async">
-      <ng-container *ngIf="(testRecordObservable$ | async)[1] as testRecord">
-        <ng-container *ngIf="(testRecordObservable$ | async)[0] as testType">
-          <vtm-test-record
-          [testRecord] = testRecord
-          [testType] = testType
-          [testTypeNumber] = this.testTypeNumber
-          [seatBeltApplicable] = this.seatBeltApplicable
-          [emissionDetailsApplicable] = this.emissionDetailsApplicable
-          [defectsApplicable] = this.defectsApplicable
-          [testSectionApplicable1] = this.testSectionApplicable1
-          [testSectionApplicable2] = this.testSectionApplicable2
+    <ng-container *ngIf="testRecord$ | async as testResultObj">
+        <ng-container *ngIf="!!testResultObj.testType && !!testResultObj.testRecord">
```

Comment:
put this `*ngIf="!!testResultObj.testType && !!testResultObj.testRecord"` on `<vtm-test-record` no need for another ng-container

Reply:
Does not work. Putting this ngIf in test record html makes accordion behaviour break (plus/minus icon doesn;t show)